src/PKSim.UI/PKSim.UI.csproj

```diff
@@ -10,6 +10,7 @@
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <NoWarn>1591</NoWarn>
     <Version Condition="'$(Version)' == ''">12.1.0</Version>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
```

Comment:
This consolidates how binary resources are serialized into the DLL. If you don't have it in csproj file then msbuild uses an older method of serializing binary data into dll and dotnet build will not work at all.

That means that no matter how you build (either through msbuild or dotnet build) you get the same result.